Riot/Modules/KeyVerification/Common/Verify/Scanning/KeyVerificationVerifyByScanningViewModel.swift

```diff
@@ -174,11 +174,12 @@ final class KeyVerificationVerifyByScanningViewModel: KeyVerificationVerifyBySca
                 self.removePendingQRCodeTransaction()
 
                 // Check due to legacy implementation of key verification which could pass incorrect type of transaction
-                if keyVerificationTransaction is MXIncomingSASTransaction {
-                    MXLog.debug("[KeyVerificationVerifyByScanningViewModel] SAS transaction should be outgoing")
-                    self.unregisterDidStateChangeNotification()
-                    self.update(viewState: .error(KeyVerificationVerifyByScanningViewModelError.unknown))
-                }
+            // TODO: Not sure about commenting this code, but MXIncomingSASTransaction has been removed
```

Comment:
@BillCarsonFr not sure how to manage this code, since we removed MXIncomingSAS stuff from the SDK

Reply:
As per https://github.com/element-hq/element-ios/pull/6775

It is ok to remove